imports/api/server/publishFunctions.js

```diff
@@ -25,24 +25,29 @@ export const getBestQualityMatch = function (animeDetailsArray) {
         return prev.sigma > curr.sigma ? prev : curr;
     });
 
-    // we define most interesting chance of winning to be the one closest to 0.5, thus start with 1 which is one of the 2 possible values for least interesting (0 being the other)
-    mostInterestingChanceOfWinning = 1;
-    bestQualityMatchObjectIndex = 0;
+    // define match quality as in equation (7) of https://www.microsoft.com/en-us/research/publication/trueskilltm-a-bayesian-skill-rating-system/
+    matchQualityFunction = function(defaultSigma, mu1, mu2, sigma1, sigma2) {
```

Comment:
Firstly, matchQualityFunction should always return a value in the range 0 to 1 (mathematically it can actually attain these values in some limits but in practice it will never reach them).

Here are some values you can test the function against:
```javascript
defaultSigma = 4
sigma1 = 2
sigma2 = 2
mu1 = 25
mu2 = 25 + Math.sqrt(Math.log(23283064365386962890625 / 121439531096594251776))
```
should return a match quality of 0.6